gnuradio-runtime/swig/message.i

```diff
@@ -22,6 +22,12 @@
 
 namespace gr {
 
+  %pythonappend message::to_string %{
+    import sys
+    if sys.version_info[0] > 2:
+      val =  val.encode("utf8", errors="surrogateescape")
```

Comment:
This is cool, but document that shit in docs/

Reply:
I'll also put the idea out there that we deprecate all PMT to_string methods. Either you want a serialization, or you don't.

Reply:
So, currently `to_string` is serializing the memory we have to a char* array?

Reply:
Not a question, but a statement.